Dev

Faire-Fuzzy opened the pull request.

OrbisAnima commented:
I think the best would be to go fixing them as we go, slowly comparing the clothes we have and the clothes we dont

Faire-Fuzzy committed:
A few minor fixes.

Fixed a minor icon issue with the fem Scorpius bot body and made a few
of the new necklace wearable as over clothing, like the collars and
such. As well as actaully giving the rosary a worn sprite.

Faire-Fuzzy committed:
A few more tweaks.

Pretty much just changes the collar/necklace clothing slot from
OCLOTHING to MASK as to not get in the way of coats and the Eros lewd
bits like breasts.

Faire-Fuzzy committed:
Revert "A few more tweaks."

This reverts commit ed4f3f38cc3c3622d60d2e28bf275f3fdb01488c.

Faire-Fuzzy committed:
Merge remote-tracking branch 'refs/remotes/ErosResearchPlatform/dev' into dev

Faire-Fuzzy committed:
Just adds a new colorable body suit to the loadout menu.

Faire-Fuzzy committed:
Little fix
